analyses/cell-type-ewings/template_notebooks/auc-singler-workflow/02-singler-results.Rmd

```diff
@@ -172,11 +172,11 @@ caret_df <- classification_df |>
 
 # get the cell types in both
 # these should be tumor normal but sometimes that's not the case
-singler_cell_types <- unique(caret_df$singler_tumor_normal) |> as.character()
-aucell_cell_types <- unique(caret_df$aucell_annotation) |> as.character()
+singler_cell_types <- unique(caret_df$singler_tumor_normal) |> as.character() |> sort()
+aucell_cell_types <- unique(caret_df$aucell_annotation) |> as.character() |> sort()
```

Comment:
Rather than sorting, let's use `setequal()`:
```suggestion
singler_cell_types <- unique(caret_df$singler_tumor_normal) |> as.character()
aucell_cell_types <- unique(caret_df$aucell_annotation) |> as.character()
```